### ISSUE TITLE
Fix Request: extend rule ImproperVariableName

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#323]
Add improper names, based on seen:
VALUE_127
VAL_127
INT_5
INTEGER_5

**Comment by jborgers** [comment]
examples:
  ```
  int VAL_127 = 127; //bad #323
    String VALUE_127 = 127; //bad #323
    int VAL_ONE_TWO_SEVEN; // bad #323
    int INT_5 = 5; //bad #323
    int INTEGER_5 = 5; //bad #323
    long long_7 = 7; //bad #323
    int NUMBER_7 = 7; //bad #323
    int NUM_7 = 7; // bad #323
    int TWENTY_EIGHT; // bad #323
```

**Comment by jborgers** [comment]
false positives:
```
class Mapper {
    String key; // good
    String value; // good
    int val; // good
}
```
